bin/pdc_client

```diff
@@ -50,6 +50,9 @@ class RequestMethod(object):
 
     @debug_request
     def get(self, data):
+        if isinstance(data, list):
+            print 'Parameter %s is error format %s, please input right format.' % (data, type(data))
```

Comment:
Should the data be a dict? If so, maybe the condition could be 
if not isinstance(data, dict):
    .........................

Should we also point out what is the correct format in error message?